iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -23,47 +30,75 @@ public enum ErrorCode
         BulkRegistryOperationFailure = 400013,
         CannotRegisterModuleToModule = 400301,
 
-        //Unauthorized - 401
+        /// <summary>
+        /// Unauthorized - 401
+        /// </summary>
         IotHubNotFound = 401001,
+
         IotHubUnauthorizedAccess = 401002,
         IotHubUnauthorized = 401003,
 
-        // Forbidden - 403
+        /// <summary>
+        /// Forbidden - 403
+        /// </summary>
         IotHubSuspended = 403001,
+
         IotHubQuotaExceeded = 403002,
```

Comment:
"The daily message quota for the IoT hub is exceeded." Just so that users know the quota is in regard to messages, not to jobs/desired property updates/methods or whatever else they think may have a quota

Reply:
https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-troubleshoot-error-403002-iothubquotaexceeded